platforms/r3-corda/configuration/roles/create/certificates/ambassador/tasks/main.yaml

```diff
@@ -18,6 +18,8 @@
     VAULT_TOKEN: "{{ vault.root_token }}"
   register: ambassador_tls_certs
   ignore_errors: yes
+  tags:
+  - notest
```

Comment:
vault should also be tested. use a docker container vault in dev mode to test Vault. Check https://hub.docker.com/_/vault?tab=description